pkg/config/config.go

```diff
@@ -134,13 +132,27 @@ func (cfg *Config) SetupLogger() {
 	ctrlrt.SetLogger(zap.New(zap.UseFlagOptions(&zapOptions)))
 }
 
+// SetAWSAccountID uses sts GetCallerIdentity API to find AWS AccountId and set
+// in Config
+func (cfg *Config) SetAWSAccountID() error {
+	// use sts to find AWS AccountId
+	session := session.Must(session.NewSession())
+	client := sts.New(session)
+	res, err := client.GetCallerIdentity(&sts.GetCallerIdentityInput{})
+	if err == nil {
+		cfg.AccountID = *res.Account
+	}
+	return err
```

Comment:
nit: handle `err != nil` first

Reply:
Better would be to wrap the error like: `fmt.Errorf("cannot get caller identity: %v", err)`